app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -228,7 +229,13 @@ def handle_unsupported_id_type(enrollment, response)
       primary_id_type: response['primaryIdType'],
       reason: 'Unsupported ID type',
     )
-    enrollment.update(status: :failed)
+    enrollment.update(status: :failed, proofed_at: proofed_at)
+
+    send_failed_email(enrollment.user, enrollment)
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_email_initiated(
+      **email_analytics_attributes(enrollment),
+      email_type: 'Failed unsupported ID type',
+    )
```

Comment:
ℹ️ We weren't previously sending emails when a user failed to proof due to an unsupported type of ID